querydsl-core/src/main/java/com/querydsl/core/types/QBean.java

```diff
@@ -73,6 +78,10 @@
         return rv.build();
     }
 
+    private static boolean isAliasSplitted(Expression<?> expr) {
```

Comment:
FactoryExpressions and GroupExpressions are compound expressions which are serialized as individual items, so you can't really alias them on the SQL/JPQL level.

The change introduced in this PR is to serialize the alias into the query, if the left hand side of the alias operation is not a FactoryExpression or GroupExpression.